Dockerfile

```diff
@@ -0,0 +1,22 @@
+# set base image (host OS)
+FROM python:3.9
```

Comment:
im running python 3.8 for some reason

Reply:
It works on Python 3.9, so everything should be OK.